js/js-fetch_star-wars/js/index.js

```diff
@@ -40,7 +40,22 @@ renderElement(firstCard);
 fetchDataAndRender();
 
 // --v-- your code below this line --v--
+// const url = ("https://swapi.dev/api/people");
 
-function fetchDataAndRender() {
-  fetch(); // ?
+async function fetchDataAndRender() {
+  try {
+    const response = await fetch("https://swapi.dev/api/people");
+    if (!response.ok) {
```

Comment:
I don't understand this if statement: I think it should say : 
`if (response.ok) {
 data.results.forEach((character) => {
      const card = Card(character);
      renderElement(card);
    });
}` => then it keep on with the if statement if the response is ok. Now it will log `response` if it is not ok

Or maybe this is what you intended?